6/README.md

```diff
@@ -0,0 +1,1078 @@
+	<tr>
+		<td> <a href="CmdDbDeleteAllDevicePipeProperty.html"> DbDeleteAllDevicePipeProperty </a> </td>
+		<td> DEVVAR_STRINGARRAY </td>
+		<td> DEV_VOID </td>
+		<td> OPERATOR </td>
+		<td> Delete all pipe properties for the specified device pipe(s) </td>
+	</tr>
+	<tr>
+		<td> <a href="CmdDbPutClassPipeProperty.html"> DbPutClassPipeProperty </a> </td>
+		<td> DEVVAR_STRINGARRAY </td>
+		<td> DEV_VOID </td>
+		<td> OPERATOR </td>
+		<td> Create/Update class pipe property(ies) in database </td>
+	</tr>
+	<tr>
+		<td> <a href="CmdDbPutDevicePipeProperty.html"> DbPutDevicePipeProperty </a> </td>
+		<td> DEVVAR_STRINGARRAY </td>
+		<td> DEV_VOID </td>
+		<td> OPERATOR </td>
+		<td> Create/Update device pipe property(ies) in database </td>
+	</tr>
+	<tr>
+		<td> <a href="CmdDbGetClassPipePropertyHist.html"> DbGetClassPipePropertyHist </a> </td>
+		<td> DEVVAR_STRINGARRAY </td>
+		<td> DEVVAR_STRINGARRAY </td>
+		<td> OPERATOR </td>
+		<td> Retrieve Tango class pipe property history </td>
+	</tr>
+	<tr>
+		<td> <a href="CmdDbGetDevicePipePropertyHist.html"> DbGetDevicePipePropertyHist </a> </td>
+		<td> DEVVAR_STRINGARRAY </td>
+		<td> DEVVAR_STRINGARRAY </td>
+		<td> OPERATOR </td>
+		<td> Retrieve device pipe property history </td>
+	</tr>
+	<tr>
+		<td> <a href="CmdDbGetForwardedAttributeListForDevice.html"> DbGetForwardedAttributeListForDevice </a> </td>
+		<td> DEV_STRING </td>
+		<td> DEVVAR_STRINGARRAY </td>
+		<td> OPERATOR </td>
+		<td> Get the list of devices using forwarded attribute(s) from specified device </td>
+	</tr>
+</table>
+
+## Database as a file
+For Device servers not able to access the Tango database (most of the time due to network route or security reason), 
+it is possible to start them using a file instead of a real database. This is done via the Device server command line option:  
+```-file=<file name>```  
+
+In this case, the Database SHALL handle the following functionality using the specified file:  
+* getting, setting and deleting class properties  
+* getting, setting and deleting device properties  
+* getting, setting and deleting class attribute properties  
+* getting, setting and deleting device attribute properties  
+
+The following set of limitations exist:  
+* no check that the same Device server is running twice
+* no Device or attribute alias name
+* in case of several Device servers running on the same host, the user must manually manage a list of already used network ports
+
+### File syntax
+The file is an ASCII file and SHALL follow a well-defined syntax with predefined keywords.
+
+Generally, comment starts with the ’#’ character and a blank line is ignored.
+
+#### Devices definition.
+DEVICE is the keyword to declare a device(s) definition sequence. The general syntax SHALL be   
+```<DS name>/<inst name>/DEVICE/<Class name>: dev1,dev2,dev3  ```
+Device(s) name can follow on next line if the last line character is ’\’
+
+#### Device property definition. 
+The general device property SHALL be  
```

Comment:
The naming convention for properties has been already defined in properties RFC-5

Reply:
Ok, I can see the confusion, this is device property definition for the file syntax only.  I have changed the heading to reflect this.  Thanks.